### DISCUSSION TITLE
Add OS Type and OS version to parameters

### DISCUSSION THREAD

**Discussion by mattdjerome** [CrowdStrike/falconpy#1023] [Ideas]
I'm using the stale sensor report to compare against our MDM and if it's not in our MDM delete it. The problem comes in when I have to do it for a specific OS. I don't see a method to do that. 

It would be great if the os architecture could be added as a parameter.

**Comment by jshcodes** [comment]
Hi @mattdjerome -

This is an excellent idea.  I've created an [issue to track this](https://github.com/CrowdStrike/falconpy/issues/1024) so we can get the functionality added.

Thank you for the suggestion! 🙇 😄
